app/views/admin/posts/_new_user_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="add-form">
+<h3>Create new user</h3>
+<%= simple_form_for(@user, url: admin_new_user_path, remote: true, validate: true, authenticity_token: true, html: { class: 'form-vertical', method: 'post', id: 'new_user' }) do |f| -%>
+  <%= hidden_field_tag 'post_id', post_id %>
+  <%= f.input :email, autofocus: true, class: 'disable-empty' %>
+  <%= f.input :name, class: 'disable-empty' %>
```

Comment:
Same here- when you look in chrome dev tools you will see that the class is not added to the input.